src/Application/Asset/StartUploadService.php

```
@@ -0,0 +1,314 @@
+                    self::DUPLICATE_CLIENT_FILE_ID_CODE,
+                    self::DUPLICATE_CLIENT_FILE_ID_MESSAGE,
+                    'clientFileId',
+                ),
+            ];
+        } else {
+            try {
+                $success = $this->initiateUpload($accountId, $file);
+            } catch (AssetDomainException $exception) {
+                $userErrors = [$this->mapDomainException($exception)];
+            }
+        }
```

PHPMD (2.15.0)

[error] The method processBatchFile uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)